cloud-data-protection/src/components/onboarding/onboarding.tsx

```diff
@@ -0,0 +1,51 @@
+import {Typography} from "@material-ui/core";
+import React, {useEffect, useState} from "react";
+import OnboardingService from "../../services/onboardingService";
+import Onboarding from "../../entities/onboarding";
+import snackbarOptions from "common/snackbar/options";
+import {useSnackbar} from "notistack";
+import OnboardingStatus from "../../entities/onboardingStatus";
+import axios, {CancelTokenSource} from "axios";
+
+const OnboardingComponent = () => {
+    const [onboarding, setOnboarding] = useState<Onboarding>();
+
+    const onboardingService = new OnboardingService();
+
+    const { enqueueSnackbar } = useSnackbar();
+
+    let cancelTokenSource: CancelTokenSource;
+
+    const fetchData = async () => {
+        cancelTokenSource = axios.CancelToken.source();
+
+        await onboardingService.get(cancelTokenSource.token)
+            .then(result => setOnboarding(result))
```

Comment:
Codacy has a fix for the issue: [Expected parentheses around arrow function argument.](https://app.codacy.com/gh/olivierbouchomsfontys/cloud-data-protection/pullRequest?prid=7219402)
```suggestion
                .then((result) => setOnboarding(result))
```